_ext/asciidoctor_toc.rb

```diff
@@ -0,0 +1,18 @@
+require 'asciidoctor'
+
+module Awestruct::Extensions
+  class AsciidoctorTOC
+    def execute(site)
+      site.pages.each do |page|
+        if page.source_path.end_with? '.adoc'
+          document = Asciidoctor.load_file page.source_path, header_footer: true, safe: :safe, attributes: 'idprefix idseparator=-'
+          page.toc = true
```

Comment:
It's just a one-line change. If you'd like, you can send the PR for it to Awestruct.

In the meantime, it's possible to monkeypatch the AsciidoctorTemplate in _ext/pipeline.rb to modify the evaluate method before the fix is available.

```ruby
require 'awestruct/handlers/template/asciidoc'

class Awestruct::Tilt::AsciidoctorTemplate
  def evaluate(scope, locals)
    @output ||= (scope.document = ::Asciidoctor.load(data, options)).convert
  end
end
```